lms/djangoapps/teams/static/teams/js/spec/edit_team_spec.js

```diff
@@ -0,0 +1,128 @@
+define([
+    'jquery',
+    'underscore',
+    'backbone',
+    'common/js/spec_helpers/ajax_helpers',
+    'teams/js/views/edit_team'
+], function ($, _, Backbone, AjaxHelpers, TeamEditView) {
+    'use strict';
+
+    describe('EditTeam', function () {
+        var teamEditView,
+            verifyValidation = function (fieldsData) {
+                _.each(fieldsData, function (fieldData) {
+                    teamEditView.$(fieldData[0]).val(fieldData[1]);
+                });
+                teamEditView.$('.create-team.form-actions .action-primary').click();
+                expect(teamEditView.$('.wrapper-msg').is(':visible')).toBeTruthy();
```

Comment:
I'm suggesting that you bind `$('.wrapper-msg')` to a variable so that you can re-use that variable instead of re-using the `'.wrapper-msg'` string. That makes it easier to refactor test failures when HTML/CSS changes.